.github/workflows/reusable-standard-its.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           ${MVN} verify -pl integration-tests -P integration-tests ${{ inputs.testing_groups }} -Djvm.runtime=${{ inputs.runtime_jdk }} -Dit.indexer=${{ inputs.use_indexer }} ${MAVEN_SKIP} -Doverride.config.path=${{ inputs.override_config_path }}
 
       - name: Debug IT
-        if: steps.run-it.conclusion == 'failure'
+        if: ${{ failure() && steps.run-it.conclusion == 'failure' }}
```

Comment:
what's the reason behind this change?

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions#failure-with-conditions Just to be explicit.